src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
I did some house cleaning ...
* ```metadata-working``` is gone, since the cleanup/development has been moved to https://github.com/mbaudis/schemas (e.g.https://github.com/mbaudis/schemas/pull/4)
* ```OntologyTerm``` reflects the current point of discussion; especially ```id``` and ```term``` seem to be stable.